temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -40,20 +40,17 @@ message CodecServerSpec {
     bool include_cross_origin_credentials = 3;
 }
 
-message GlobalNamespaceSpec {
-    // The regions where the namespace is (or will be) located.
-    // Optional, default is empty.
-    repeated string passive_regions = 1;
-}
-
 message NamespaceSpec {
     // The name to use for the namespace.
     // This will create a namespace that's available at '<name>.<account>.tmprl.cloud:7233'.
     // The name is immutable. Once set, it cannot be changed.
     string name = 1;
-    // The region where the namespace is (or will be) active.
-    // The region is immutable. Once set, it cannot be changed.
-    string region = 2;
+    // The regions where the namespace should be hosted.
+    // Specifying more than one region makes the namespace a Global Namespace, which is currently a preview only feature with restricted access.
```

Comment:
Specifying more than one region makes the namespace "global", which is currently a preview only feature with restricted access.